pennylane/templates/tensornetworks/ttn.py

```diff
@@ -56,33 +56,21 @@ def compute_indices(wires, n_block_wires):
     n_wires = 2 ** (int(np.log2(len(wires) / n_block_wires))) * n_block_wires
     n_layers = int(np.log2(n_wires // n_block_wires)) + 1
 
-    layers = [
```

Comment:
Basically just switching lists to tuples.

But I couldn't help myself and had to fix this block of code.